welcome.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+console.log();
+console.log('*************************************************************************');
+console.log('*                                                                       *');
+console.log('* To get started, please create a project. Type the following to begin: *');
+console.log('*                                                                       *');
+console.log('* twilio login -p <PROJECT NAME>                                        *');
```

Comment:
should we add another line to also teach them they can alternatively use `twilio project:add <project name>`